pkgs/applications/misc/haunt/default.nix

```diff
@@ -0,0 +1,61 @@
+{ lib
+, stdenv
+, fetchurl
+, guile
+, guile-commonmark
+, guile-reader
+, makeWrapper
+, pkg-config
+}:
+
+stdenv.mkDerivation rec {
+  pname = "haunt";
+  version = "0.2.4";
+
+  src = fetchurl {
+    url = "https://files.dthompson.us/${pname}/${pname}-${version}.tar.gz";
+    hash = "sha256-zOkICg7KmJJhPWPtJRT3C9sYB1Oig1xLtgPNGe0n3xQ=";
+  };
+
+  nativeBuildInputs = [
+    makeWrapper
+    pkg-config
+  ];
+  buildInputs = [
+    guile
+  ];
+  propagatedBuildInputs = [
+    guile-commonmark
+    guile-reader
+  ];
```

Comment:
IIRC the makeWrapper should be enough and propagatedBuildInputs shouldn't be used anymore if not absolutely required.

Reply:
Well, I used the guix expression for haunt as a guide here.